### DISCUSSION TITLE
Need an example for excluded_apps.txt as MSI parameter

### DISCUSSION THREAD

**Discussion by oXydus** [Romanitho/Winget-AutoUpdate#733] [Q&A] [answered]
I'm trying to silent deploy WAU.msi with parameters but struggling to copy excluded_apps.txt from the installation folder to Program Files (like it happens when using GUI installer).
I put excluded_apps.txt in the same folder as WAU.msi but this file does not copy to Program Files folder.
I used the following line code:
msiexec.exe /i WAU.msi /qn DONOTRUNONMETERED=0 RUN_WAU=YES USERCONTEXT=1 STARTMENUSHORTCUT=1 UPDATESINTERVAL=Daily
If I add USEWHITELIST=1 LISTPATH=.\ (or other path) - I receive an error that list does not exists.
In README.md there is an example for URI -ListPath, but how to use LISTPATH to copy excluded_apps.txt from the installation folder to Program Files folder?
Thank you

**Comment by Romanitho** [comment]
Hi,
The ListPath is there to set a path to a regularly based list.
If you want to configure a static app list silently, you just have to put the file in the same folder as wau.msi. It will detect the file and copy it to the Program Files folder.

**Reply by oXydus** [reply]
I did like you said, but the file does not copy. Can you test this script with a blank excluded_apps.txt and WAU.msi in the same folder?
msiexec.exe /i "D:\WAU.msi" /qn DONOTRUNONMETERED=0 RUN_WAU=YES USERCONTEXT=1 STARTMENUSHORTCUT=1 UPDATESINTERVAL=Daily

**Reply by Romanitho** [reply, marked as answer]
that's because your don't run WAU.msi from the current directory.

**Reply by Romanitho** [reply]
but I can try to find the correct variable to get the msi folder location. This way, it should always use the folder where WAU.msi is, instead of current folder.

**Reply by oXydus** [reply]
Yes, thank you! Amazing work!
